config/policies/default.cedar

```diff
@@ -44,4 +44,21 @@ forbid (
     action == Access::Action::"Approve",
     resource
 )
-when { principal == resource.principal };
\ No newline at end of file
+when { principal == resource.principal };
+
+// Allow admins to read all resources
+permit (
+    principal,
+    action in CF::Admin::Action::"Read",
+    resource
+);
+
+
+permit (
+    principal ==
+        CF::User::<Replace with your own user id from CF web console>,
+    action ==
+        CF::RPC::Action::"commonfate.control.integration.v1alpha1.IntegrationService/ListSyncsForIntegration",
+    resource ==
+        CF::Service::"ControlPlane",
+);
```

Comment:
This can be removed, in new versions the ListSyncsForIntegration action is correctly grouped in the `CF::Admin::Action::"Read"` group

```suggestion
```